src/index.ts

```diff
@@ -150,6 +151,145 @@ app.get("/api/users/:id", async (req, res) => {
   }
 });
 
+
+/**
+ * @openapi
+ * /api/users: 
```

Comment:
It's good to add the parameter to the uri of the endpoint - without it, the GET endpoint documentation on swagger page got broken 😄 

```suggestion
 * /api/users/{id}: 
```

Reply:
Got it, thanks